src/data_model/task.rs

```diff
@@ -1,17 +1,15 @@
-use::uuid::Uuid;
-use::chrono::{DateTime, Local};
-struct TimeInteval {
-    start_time: DateTime<Local>,
-    end_time: DateTime<Local>,
-}
+use serde::{Deserialize, Serialize};
 
-struct Task {
-    device_id: Uuid,
-    effect: i16,
-    duration: i32,
-    interval: TimeInteval,
+#[derive(Serialize, Deserialize)]
+pub struct Timespan {
```

Comment:
```suggestion
pub struct TimePeriod {
```
Rename to TimePeriod or something else, TimeSpan is https://learn.microsoft.com/en-us/dotnet/api/system.timespan?view=net-8.0, plz don't use that name

Reply:
This is exactly the same